test/e2e/etcdcluster_test.go

```diff
@@ -134,6 +134,20 @@ var _ = ginkgo.Describe("etcdcluster", func() {
 			})
 			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
 		})
+
+		ginkgo.It("kstone should generate etcdinspection/alarm resources", func() {
```

Comment:
we should enable the alarm feature, otherwise this test case will always be true.